template/interpolate/aws/secretsmanager/secretsmanager.go

```diff
@@ -0,0 +1,97 @@
+// Package secretsmanager provide methods to get data from
+// AWS Secret Manager
+package secretsmanager
+
+import (
+	"encoding/json"
+	"errors"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/secretsmanager"
+	"github.com/aws/aws-sdk-go/service/secretsmanager/secretsmanageriface"
+)
+
+// SecretsManager returns a representation of the Secrets Manager API
+func (c *Client) SecretsManager() secretsmanageriface.SecretsManagerAPI {
```

Comment:
Is this function needed?

Reply:
Good catch, it is not used anymore removing it.